### PR TITLE
Update Messenger channel docs for blocked buy buttons

### PR DESCRIPTION
**Pull request by spasiu**
@jpjoyal @mspensieri @chloepouprom 

Changed docs to reflect what's currently in prod

### CODE REVIEW

**Comment by mspensieri** [comment]
👍

File: source/includes/messaging_channels/_messenger.md [hunk 1]
```diff
@@ -15,4 +15,4 @@ Once your Facebook account is connected to Smooch, select the Facebook page you
 
 Action buttons work as expected. A `link` action button will bring the user to the specified URL while a `postback` action button will trigger any [webhook](/rest#webhooks) listening to the `postback` trigger.
 
-Payment requests (action buttons of type `buy`) will turn into link buttons which bring the user to a checkout page where payment information can be entered.
+Because of new Facebook terms of service, payment requests (action buttons of type `buy`) cannot currently be sent to Messenger.
```
Comment (jpjoyal) [inline source/includes/messaging_channels/_messenger.md]: link to the term section here would be great

Comment (jpjoyal) [inline source/includes/messaging_channels/_messenger.md]: you can use this https://developers.facebook.com/policy/#messengerplatform

**Comment by jpjoyal** [comment]
👍

**Pull request merged**